src/plugins/options/option/option.ts

```diff
@@ -6,72 +6,60 @@ import {ENTER} from '@exadel/esl/modules/esl-utils/dom/keys';
 
 /** Config used to create options. */
 export type OptionConfig = {
-    /** Attribute to toggle. */
-    attribute: string;
-    /** Location of option's icon. */
-    iconUrl: string;
-    /** Callback to indicate if option should be rendered. */
-    canActivate?: () => boolean;
+  /** Attribute to toggle. */
+  attribute: string;
+  /** Callback to indicate if option should be rendered. */
+  canActivate?: () => boolean;
 };
 
 /** Custom element to toggle {@link UIPRoot} attributes. */
 export class UIPOption extends ESLBaseElement {
-    static is = 'uip-option';
-    @attr() public attribute: string;
-    @boolAttr() public active: boolean;
-    protected icon: ESLImage;
+  static is = 'uip-option';
+  @attr() public attribute: string;
+  @boolAttr() public active: boolean;
 
-    static create(optionConfig: OptionConfig): UIPOption {
-        const option = document.createElement('uip-option') as UIPOption;
-        option.icon = document.createElement('esl-image');
-        option.icon.mode = 'inner-svg';
-        option.icon.dataset.src = optionConfig.iconUrl;
-        option.setAttribute('attribute', optionConfig.attribute);
+  static create(optionConfig: OptionConfig): UIPOption {
+    const option = document.createElement('uip-option') as UIPOption;
+    option.setAttribute('attribute', optionConfig.attribute);
+    return option;
+  }
 
-        return option;
-    }
+  protected connectedCallback() {
+    super.connectedCallback();
+    this.classList.add(`${this.attribute}-option`);
+    this.tabIndex = 0;
+    this.bindEvents();
+  }
 
-    protected connectedCallback() {
-        super.connectedCallback();
-        this.classList.add(`${this.attribute}-option`);
-        this.tabIndex = 0;
-        this.bindEvents();
-        this.render();
-    }
+  protected bindEvents() {
+    this.addEventListener('click', this._onClick);
```

Comment:
Just a note:
You can cinsider upgrade ESL to 4th beta, then you can simplify a lot all listener managemant
CC: @yadamskaya